.env

```diff
@@ -0,0 +1,3 @@
+DATABASE_URL = mongodb+srv://lawallarkson:$vMv8a3Di9tpSQY@halaqaat.sd2ol.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
```

Comment:
Glad you use Environment Variable. You should add .env to the .ignore file to make sure you don't commit a Sensitive file like this.

Reply:
I believe I added it to .gitignore 
My bad perhaps it didn't effect